CHANGELOG.md

```diff
@@ -25,6 +25,7 @@ x.y.z Release notes (yyyy-MM-dd)
 * Fix a race condition which could result in "operation cancelled" errors being delivered to async open callbacks rather than the actual sync error which caused things to fail ([PR #5968](https://github.com/realm/realm-core/pull/5968), since the introduction of async open).
 * Bootstraps will not be applied in a single write transaction - they will be applied 1MB of changesets at a time, or as configured by the SDK ([#5999](https://github.com/realm/realm-core/pull/5999), since v10.27.0).
 * Fix database corruption and encryption issues on apple platforms, reported in several bugs listed in the PR. ([PR #5993](https://github.com/realm/realm-core/pull/5993), since v10.21.1)
+* Add extra check when allocating string when receiving data in a changestream [PR #8039](https://github.com/realm/realm-swift/pull/8039).
```

Comment:
Can we reformulate this from the perspective of users? How would they experience this bug?